jfr-streaming/src/test/java/io/opentelemetry/contrib/jfr/metrics/AbstractMetricsTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.jfr.metrics;
+
+import com.sun.tools.attach.VirtualMachine;
+import io.opentelemetry.sdk.metrics.SdkMeterProvider;
+import io.opentelemetry.sdk.metrics.data.MetricData;
+import org.junit.jupiter.api.BeforeAll;
+import io.opentelemetry.sdk.metrics.testing.InMemoryMetricReader;
+import io.opentelemetry.sdk.testing.assertj.metrics.MetricAssertions;
+import io.opentelemetry.sdk.testing.assertj.metrics.MetricDataAssert;
+import java.util.Collection;
+import java.util.function.Consumer;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+public class AbstractMetricsTest {
+
+  static SdkMeterProvider meterProvider;
+  static InMemoryMetricReader metricReader;
```

Comment:
Note, post my latest PR you'll be able to create one of these that asks for DELTA on all metrics.